src/pages/en/guides/integrations-guide.md

```diff
@@ -159,6 +104,16 @@ integrations: [
 ]
 ```
 
+## Finding More Integrations
+
+You can find many integrations developed by the community in the [Astro Integrations Directory](https://astro.build/integrations/). Follow links there for detailed usage and configuration instructions.
+
+:::note[Experimental status]
+**To enable third-party integrations:** Run Astro with the `--experimental-integrations` CLI flag, or include `experimental: { integrations: true }` in your Astro config file.
```

Comment:
May already be tracked elsewhere, but I'm pretty sure we'll be getting rid of the `experimental` flag as part of the 1.0 release candidate